assembly/hdp2.6.3/build/var/lib/ambari-server/resources/common-services/SPLICEMACHINE/2.5.1/service_advisor.py

```diff
@@ -0,0 +1,248 @@
+        if property not in core_site or core_site[property] != desired_value:
+          putCoreSiteProperty(property, desired_value)
+
+    # Update hbase-site properties in hbase-site
+    if "hbase-site" in services["configurations"]:
+      hbase_site = services["configurations"]["hbase-site"]["properties"]
+      putHbaseSitePropertyAttributes = self.putPropertyAttribute(configurations, "hbase-site")
+      putHBaseSiteProperty = self.putProperty(configurations, "hbase-site", services)
+      for property, desired_value in self.getHBaseSiteDesiredValues().iteritems():
+        if property not in hbase_site or hbase_site[property] != desired_value:
+          putHBaseSiteProperty(property, desired_value)
+
+
+    # Update HIVE_AUX_JARS_PATH for hive
+    if "hive-env" in services["configurations"]:
+        hive_env = services['configurations']['hive-env']["properties"]
+        if "content" in hive_env:
+            hive_content = hive_env["content"]
+            HIVE_SPLICE_PATH = "export HIVE_AUX_JARS_PATH=${HIVE_AUX_JARS_PATH}:/var/lib/splicemachine/"
+            if "splicemachine" not in hive_content:
+                HIVE_SPLICE_PATH = "#Add Splice Jars to HIVE_AUX_JARS_PATH\n" + HIVE_SPLICE_PATH
+                content = "\n\n".join((content, HIVE_SPLICE_PATH))
+                putHiveEnvProperty = self.putProperty(configurations, "hive-env", services)
+                putHiveEnvProperty("content", content)
+
+    # Update spark-defaults for spark
+    if "spark2-env" in services["configurations"]:
+        spark2_env = services['configurations']['spark2-env']["properties"]
+        splice_driver_lib = "export spark.driver.extraLibraryPath=${spark.driver.extraLibraryPath}:" + splice_jars + "\n"
+        splice_executor_lib = "export spark.executor.extraLibraryPath=${spark.executor.extraLibraryPath}:" + splice_jars + "\n"
+        splice_path = "\n".join((splice_driver_lib,splice_executor_lib))
+        if 'content' in spark2_env:
+          spark_content = spark2_env["content"]
+          if "splicemachine" not in spark_content:
+            spark_content = "\n\n".join((spark_content,splice_path))
+            putSparkProperty = self.putProperty(configurations, "spark2-env", services)
+            putSparkProperty("content",spark_content)
+
+   #update zookeeper configs
+    if 'zoo.cfg' in services['configurations']:
+      zoo_cfg = services['configurations']['zoo.cfg']["properties"]
+      print(zoo_cfg),zoo_cfg
+      putZooProperty = self.putProperty(configurations, "zoo.cfg", services)
+      putZooProperty('maxClientCnxns',0)
+      putZooProperty('maxSessionTimeout',120000)
+
+    if 'yarn-site' in services['configurations']:
+      yarn_site = services['configurations']['yarn-site']["properties"]
+      putYarnSiteProperty = self.putProperty(configurations,"yarn-site",services)
+      yarn_application_classpath = yarn_site['yarn.application.classpath'] + ',/var/lib/splicemachine/lib/'
```

Comment:
We wanted to remove splice from yarn's application classpath, right?

Reply:
That is correct.